lib/bundle/dumper.rb

```diff
@@ -20,6 +20,7 @@ def self.dump_brewfile
       content << cask_before_formula
       content << BrewDumper.dump
       content << cask_after_formula
+      content << MacAppStoreDumper.dump
```

Comment:
Personally, I would ask a flag to disable this. Because not everyone want to use `brew bundle` to manage apps.